custom/enikshay/tasks.py

```diff
@@ -193,9 +193,9 @@ def get_adherence_schedule_start_date(self):
         if not raw_date:
             return None
         if parse_datetime(raw_date):
-            return parse_datetime(raw_date)
+            return (parse_datetime(raw_date)).replace(tzinfo=pytz.utc)
```

Comment:
Actually both `episode.adherence_schedule_start_date` and `adherence_date` are just IST date objects (example [case](https://india.commcarehq.org/a/enikshay/reports/case_data/2822c551-01c8-47d4-b187-c87651e807e0/)). So I have changed the code to treat them as date objects